superset-frontend/src/dashboard/components/PropertiesModal/index.tsx

```diff
@@ -123,23 +123,35 @@ const PropertiesModal = ({
   };
 
   const loadAccessOptions = useCallback(
-    (accessType = 'owners', input = '', page: number, pageSize: number) => {
+    (
+      accessType = 'owners',
+      input = '',
+      checkActive: boolean = true,
+      page: number,
+      pageSize: number,
+    ) => {
       const query = rison.encode({
         filter: input,
         page,
         page_size: pageSize,
       });
       return SupersetClient.get({
         endpoint: `/api/v1/dashboard/related/${accessType}?q=${query}`,
-      }).then(response => ({
-        data: response.json.result
-          .filter((item: { extra: { active: boolean } }) => item.extra.active)
```

Comment:
maybe we can check if item.extra.active attribute is there and if yes then leverage that otherwise not

Reply:
sure, ok.
will update it in a bit.